bin/recordium

```diff
@@ -28,6 +28,7 @@ parent_dir = os.path.dirname(os.path.dirname(os.path.realpath(sys.argv[0])))
 if os.path.basename(parent_dir) == 'recordium':
     # inside the project!!
     sys.path.insert(0, parent_dir)
+    os.chdir(parent_dir) #to allow use of relative paths
```

Comment:
comment should be more separated (two blanks from the code, one blank after the hash)